modules/preloader/browser.js

```diff
@@ -2,16 +2,14 @@
 @module preloader browser
 */
 require('./include/common')('browser');
-const electron = require('electron');
-const ipc = electron.ipcRenderer;
-const shell = electron.shell;
+const { ipcRenderer: ipc, shell } = require('electron');
 const mist = require('../mistAPI.js');
 const BigNumber = require('bignumber.js');
 const ipcProviderWrapper = require('../ipc/ipcProviderWrapper.js');
 const Web3 = require('web3');
-require('../getFavicon.js');
-require('../getMetaTags.js');
-require('../openExternal.js');
+require('./include/getFavicon.js');
+require('./include/getMetaTags.js');
+require('./include/openExternal.js');
```

Comment:
This is rather weird, not sure how this (https://github.com/ethereum/mist/commits/develop/modules/preloader/browser.js) happened.